### ISSUE TITLE
Incompatible with Wagtail 2.13

### ISSUE THREAD

**Issue by mixxorz** [ixc/wagtail-instance-selector#11]
Reporting an incompatibility with the soon-to-be-released Wagtail 2.13. I've tried upgrading the Wagtail project I'm working on right now and I'm hitting this error on startup:

```
telepath.UnpackableTypeError: don't know how to pack object: <function InstanceSelectorBlock.get_instance_selector_icon at 0x7f76e45b44c0
```

It has to do with the new way form field widgets are rendered on the front-end.

I've only just found this out and trying to see if I could come up with a solution.

**Comment by markfinger** [comment]
Thanks for the report.

Looks like it probably needs some python hooks added to enable js init. See: https://wagtail.github.io/telepath/api_reference/

**Comment by markfinger** [comment]
It's probably simplest to wait for 2.13 to release: https://docs.wagtail.io/en/stable/releases/index.html

Hopefully wagtail core will either shoe-horn in some backwards compatibility or they'll document a migration path.

**Comment by markfinger** [comment]
Fixed in `wagtail-instance-selector==2.1.0`.
